autouam/config/settings.py

```diff
@@ -244,6 +244,16 @@ class Settings(BaseSettings):
         "extra": "ignore",
     }
 
+    @model_validator(mode="before")
+    @classmethod
```

Comment:
No reason for this to be a classmethod as the class it never accessed, can be staticmethod I guess